lock/src/main/java/com/auth0/android/lock/views/ClassicPanelHolder.java

```diff
@@ -320,6 +348,8 @@ public void onClick(View v) {
             event = changePwdForm.submitForm();
         } else if (customFieldsForm != null) {
             event = customFieldsForm.submitForm();
+        } else if (mfaCodeFormView != null) {
```

Comment:
In this case, the first call returning an event should break and fire it. If I use NullObjects here for the forms, I'll avoid the null check of the form but I'll still need to check for a valid event.